hack/verify-test-code.sh

```diff
@@ -35,6 +35,10 @@ do
     then
         errors_expect_no_error+=( "${file}" )
     fi
+    if grep -E "Expect\(err\)\.To\(gomega\.BeNil\(\)\)" "${file}" > /dev/null
```

Comment:
cool!

Would be nice to have the same for this: https://github.com/kubernetes/kubernetes/pull/93214/files